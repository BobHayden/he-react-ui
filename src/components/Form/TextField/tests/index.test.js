import createTestContext from 'react-cosmos-test/enzyme'
import fixture from '../fixtures/Default.fixture'

const { mount, getWrapper } = createTestContext({ fixture })

beforeEach(mount)

test('<TextField /> rendered correctly with fixture', () => {
  let wrapper = getWrapper()
  expect(wrapper.find('label').length).toEqual(2)
  expect(wrapper.find('input').length).toEqual(1)

  expect(wrapper).toMatchSnapshot()
})
