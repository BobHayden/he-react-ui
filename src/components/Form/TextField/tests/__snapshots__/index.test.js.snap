// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<TextField /> rendered correctly with fixture 1`] = `
<TextField
  description="Optional descriptive label"
  disabled={false}
  id="demo"
  inline={false}
  label="First Name"
  marker={false}
  name="demo"
  onBlur={[Function]}
  onChange={[Function]}
  onFocus={[Function]}
  validateFunc={[Function]}
  value=""
>
  <div
    className=""
  >
    <div>
      <label
        className=""
        htmlFor="demo"
      >
        First Name
      </label>
      <input
        disabled={false}
        id="demo"
        name="demo"
        onBlur={[Function]}
        onChange={[Function]}
        onFocus={[Function]}
        value=""
      />
    </div>
    <label
      htmlFor="demo"
    >
      Optional descriptive label
    </label>
  </div>
</TextField>
`;
